<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KursovaRemakeClass</class>
 <widget class="QMainWindow" name="KursovaRemakeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KursovaRemake</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="chooseEngine_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose engine type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ICEngine_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ICEngine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EngineButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Engine</string>
    </property>
   </widget>
   <widget class="QPushButton" name="JetEngine_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JetEngine</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="engineName_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the engine name</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="enginePower_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter HORSEPOWEER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sortBy_label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sort by</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_name">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_power">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HoursePower</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_type">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Engine type</string>
    </property>
   </widget>
   <widget class="QLabel" name="sleeping_label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sleeping</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>260</y>
      <width>81</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ready_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>READY</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>149</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Power</string>
     </property>
     <property name="font">
      <font>
       <family>Cascadia Code</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionNew"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="KursovaRemake.qrc"/>
 </resources>
 <connections/>
</ui>
